src/DatasetTree.ts

```diff
@@ -63,6 +62,52 @@ export class DatasetTree extends ZoweTreeProvider implements IZoweTree<IZoweData
         this.treeView = vscode.window.createTreeView("zowe.explorer", {treeDataProvider: this});
     }
 
+    /**
+     * Rename data sets
+     *
+     * @export
+     * @param {ZoweDatasetNode} node - The node
+     * @param {DatasetTree} datasetProvider - the tree which contains the nodes
+     */
+    public async rename(node: IZoweDatasetTreeNode) {
```

Comment:
Thanks for keeping this PR at one command.
And for adding the placeholders for the other functions. 👍